source/quick-start-sinatra/view-data.txt

```diff
@@ -0,0 +1,150 @@
+.. _mongoid-quick-start-sinatra-view-data:
+
+=================
+View MongoDB Data
+=================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create an application file
+      
+      At the root level of your project, create a file called ``app.rb``.
+      Paste the following contents into the ``app.rb`` file to
+      load the necessary gems and your configuration file:
+
+      .. code-block:: ruby
+
+         require 'sinatra'
+         require 'mongoid'
+         
+         Mongoid.load!(File.join(File.dirname(__FILE__), 'config', 'mongoid.yml'))
+
+   .. step:: Create a data model
+
+      In the ``app.rb`` file, create a model called ``Restaurant``
+      to represent data from the sample ``restaurants`` collection in
+      your MongoDB database:
+
+      .. code-block:: ruby
+
+         class Restaurant
+           include Mongoid::Document
+         
+           field :name, type: String
+           field :cuisine, type: String
+           field :borough, type: String
+         
+         end
+
+   .. step:: Generate a view
+
+      To display your data in a specified way by using HTML, you can
+      create a **view**.
```

Comment:
S: since this is a [procedure step](https://www.mongodb.com/docs/meta/style-guide/style/tasks/#use-imperative-sentences), I'd change the verb tense:
```suggestion
      Create a **view** to display your data in a specified way by using HTML.
```